oiva-core/src/test/java/fi/minedu/oiva/backend/core/service/MuutospyyntoServiceTest.java

```diff
@@ -538,6 +539,36 @@ public void testFilterOldMuutokset() {
         assertEquals(1, maaraykset.size());
     }
 
+    @Test
+    public void testValidAsianumero() {
+        assertFalse(service.validAsianumero("VN/1234/123"));
+        assertFalse(service.validAsianumero("VN/1456/1234T"));
+        assertFalse(service.validAsianumero("VN//1234"));
+        assertFalse(service.validAsianumero("VN/1234567/1234"));
```

Comment:
Keskimmäinen kenttä voi olla enintään 9 numeroa pitkä (myös CSCOIVA-1291 descriptionin mukaisesti). Enintään 6 merkkiä on vanhentunut määritys.

Reply:
Korjattu testi sekä toteutus

Reply:
Kannattaa myös testata käyttöliittymän tai integraatiotestien kautta, riittääkö nykyisen asianumero-kentän pituus näin pitkien merkkijonojen tallennukseen